<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prototipo Facturacion Frisby</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="8" column="1">
       <widget class="QLabel" name="lb_restaurante">
        <property name="text">
         <string>Restaurante</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_seleccionar">
        <property name="text">
         <string>Seleccionar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Proveedor de red</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_nuevo_restaurante">
        <property name="text">
         <string>Nuevo Restaurante</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QComboBox" name="cb_anhos"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lb_ruta">
        <property name="text">
         <string>Ruta del archivo</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QComboBox" name="cb_meses"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_operadores">
        <property name="text">
         <string>Operadores de red</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="lb_anho">
        <property name="text">
         <string>Año</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lb_nombre_restaurante">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Restaurante</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="4">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="lb_nombre_operador">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lb_mes">
        <property name="text">
         <string>Mes</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QPushButton" name="btn_consultar_todo">
        <property name="text">
         <string>Consultar Todo</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QPushButton" name="btn_consultar_mes">
        <property name="text">
         <string>Consultar Mes</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="btn_analizar">
        <property name="text">
         <string>Analizar</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="cb_restaurantes"/>
      </item>
      <item row="10" column="1">
       <widget class="QPushButton" name="btn_consultar_anho">
        <property name="text">
         <string>Consultar Año</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="cb_operador"/>
      </item>
      <item row="6" column="0" rowspan="2" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_matricula">
        <property name="text">
         <string>Matricula</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLineEdit" name="le_matricula">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_operador</tabstop>
  <tabstop>btn_operadores</tabstop>
  <tabstop>btn_nuevo_restaurante</tabstop>
  <tabstop>btn_seleccionar</tabstop>
  <tabstop>btn_analizar</tabstop>
  <tabstop>cb_meses</tabstop>
  <tabstop>cb_restaurantes</tabstop>
  <tabstop>cb_anhos</tabstop>
  <tabstop>btn_consultar_mes</tabstop>
  <tabstop>btn_consultar_anho</tabstop>
  <tabstop>btn_consultar_todo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
